BuyBitcoin/settings.py

```diff
@@ -65,8 +68,19 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'authentication.plaid_middleware.PlaidMiddleware',
+    'security.middleware.DoNotTrackMiddleware',
+    'security.middleware.ContentNoSniff',
+    # 'security.middleware.LoginRequiredMiddleware',
 ]
 
+MIDDLEWARE_ClASSES = (
+    'security.middleware.LoginRequiredMiddleware',
+    'security.middleware.XFrameOptionsMiddleware',
+    'MandatoryPasswordChangeMiddleware',
+    'SessionExpiryPolicyMiddleware',
+    'StrictTransportSecurityMiddleware',
+    )
+
```

Comment:
Middleware and Middleware classes can not coexist I think

Reply:
you're right. I made modifications to the package but im realizing now that because thats stored in my venv that wont work. I'll fix and patch.